src/brasil/gov/agenda/upgrades/v4104/profile/types/Agenda.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<object name="Agenda" meta_type="Dexterity FTI"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
```

Comment:
you're not translating anything; you can safely remove `xmlns:i18n` and `i18n:domain`.